sdk/servicebus/service-bus/src/receivers/sessionReceiver.ts

```diff
@@ -253,9 +260,15 @@ export class SessionReceiverImpl<ReceivedMessageT extends ReceivedMessage | Rece
    */
   async renewSessionLock(options?: OperationOptions): Promise<Date> {
     this._throwIfReceiverOrConnectionClosed();
-
+    if (!this._messageSession) {
+      const error = new MessagingError(
+        `The session lock has expired on the session with id ${this.sessionId}.`
+      );
+      error.code = "SessionLockLostError";
+      error.retryable = false;
```

Comment:
I think for amqp errors, we do this a little differently. Using the `SenderBusyError` as reference

```js
const amqpError: AmqpError = {
            condition: ErrorNameConditionMapper.SessionLockLostError,
            description: "error message goes here"
          };
throw(translate(amqpError));
```

Reply:
Done!